Packages/com.unity.inputsystem/InputSystem/Controls/InputControlLayout.cs

```diff
@@ -2037,6 +2082,34 @@ public InternedString GetRootLayoutName(InternedString layoutName)
                 return layoutName;
             }
 
+            public InternedString FindLayoutThatIntroducesControl(InputControl control, Cache cache)
```

Comment:
Should we have a unit test for this method?

Reply:
It's an internal API. We have a test that covers the public API that uses this (only one so far).